Bio/Entrez/Parser.py

```diff
@@ -42,6 +42,7 @@
 from xml.parsers import expat
 from io import BytesIO
 import xml.etree.ElementTree as ET
+from xml.sax.saxutils import escape
```

Comment:
You don't use this escape module in this file, do you? i.e. Can it be removed.

(This also avoids the potential confusion with an ``__init__`` argument also called ``escape``)

Reply:
escape is a function.

Reply:
I see it now,

```python
    def characterDataHandlerEscape(self, content):
        content = escape(content)
        self.consumer.consume(content)
```

Sorry, false alarm.